samples/highcharts/accessibility/custom-dynamic/demo.html

```diff
@@ -2,5 +2,11 @@
 <script src="https://code.highcharts.com/modules/exporting.js"></script>
 <script src="https://code.highcharts.com/modules/export-data.js"></script>
 <script src="https://code.highcharts.com/modules/accessibility.js"></script>
-<div id="container"></div>
-<button id="stop">Stop updating live data</button>
+<figure class="highcharts-figure">
+    <div id="container"></div>
```

Comment:
It is difficult to get beyond "Live updating data", because the screen reader resets the focus of the hidden description with each point update. If only the text would be updated instead of the DOM element, the issue might not exist. The stop button is unfortunately beyond the chart.

Reply:
Dynamic updates are a known issue and suggestions are noted down. For now, I will change the button to "Start Animation" initially. This will give the user a bit more context about the chart before pressing play.